### PR TITLE
fix(emotion): Autolabel name sanitisation

### PR DESCRIPTION
**Pull request by Codex-**
Pending #155 

This ports over the label sanitisation behaviour from emotin babel plugin, which sanitises the label input.

### CODE REVIEW

File: packages/emotion/transform/src/lib.rs [hunk 1]
```diff
@@ -240,12 +254,13 @@ impl<C: Comments> EmotionTransformer<C> {
                 .unwrap_or_else(|| "[local]".to_owned())
         );
         if let Some(current_context) = &self.current_context {
-            label = label.replace("[local]", current_context);
+            label = label.replace("[local]", &self.sanitize_label_part(current_context));
             if let Some(filename) = self.filename.as_ref() {
-                label = label.replace("[filename]", filename);
+                // label = label.replace("[filename]", &self.sanitize_label_part(&filename));
```
Comment (aburd) [inline packages/emotion/transform/src/lib.rs:259]: probably don't need this

Comment (Codex-) [reply packages/emotion/transform/src/lib.rs:259]: Ah yup my mistake, can't push any changes as I'm afk til Tuesday 😔

Comment (Codex-) [reply packages/emotion/transform/src/lib.rs]: Removed the commented code

**Comment by Codex-** [comment]
Rebased on latest `main`, ready for review :)

**Review by kdy1 — APPROVED** [review]
Thank you!

**Pull request merged**